client/src/components/Auth/Register/__tests__/Register.test.jsx

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import { makeMockStore } from "../../../../test/testUtils";
+import { cleanup, fireEvent, wait } from "@testing-library/react";
+import { renderWithRouter } from "../../../../test/routerTestUtils";
+import { Redirect as MockRedirect } from "react-router-dom";
+import API from "../../../../lib/api";
+import Register from "../Register";
+
+const mockResponse = {
+  success: true,
+  token: "Bearer 5cCI6IkpXVCJ9.oxNTkwNTI4NjA3fQ.1Nd3GGE8"
+};
+
+const defaultProps = {
+  auth: {
+    isAuthenticated: false,
+    user: {},
+    loading: false
+  }
+};
+
+jest.mock("react-router-dom", () => {
+  const original = jest.requireActual("react-router-dom");
+  return {
+    ...original,
+    Redirect: jest.fn().mockImplementation(() => null)
+  };
+});
+
+jest.mock("../../../../lib/api", () => ({
+  post: jest.fn(() => Promise.resolve())
+}));
+
+afterEach(cleanup);
+
+let store;
+beforeEach(() => {
+  store = makeMockStore(defaultProps);
+});
+
+describe("Register rendering", () => {
+  test("Register renders without error", () => {
+    const { getByTestId } = renderWithRouter(<Register store={store} />);
+
+    expect(getByTestId("register")).toBeTruthy();
```

Comment:
or `toExist`?

Reply:
this will get updated to `queryByTestId` when i refactor things. then truthy will be a more accurate assertion.